<?xml version="1.0" encoding="utf-8"?>

<launch>
  <arg name="joy_config" default="ps3" />
  <arg name="joy_dev" default="/dev/input/by-id/usb-DragonRise_Inc._Generic_USB_Joystick-joystick" />
  <arg name="config_filepath" default="$(find teleop_twist_joy)/config/$(arg joy_config).config.yaml" />
  <arg name="joy_topic" default="joy" />
  <arg name="cmd_vel_topic" default="/aww/aww_velocity_controller/cmd_vel" />
  
  <node pkg="joy" type="joy_node" name="joy_node">
    <param name="dev" value="$(arg joy_dev)" />
    <param name="deadzone" value="0.3" />
    <param name="autorepeat_rate" value="20" />
    <remap from="joy" to="$(arg joy_topic)" />
  </node>

  <node pkg="aww_control" name="aww_teleop" type="aww_teleop.py">
    <remap from="aww/cmd_vel" to="$(arg cmd_vel_topic)" />
  </node>

</launch>